Gemfile

```diff
@@ -18,7 +18,9 @@ gem 'tzinfo-data', platforms: %i[ windows jruby ]
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', require: false
-
+group :development, :test do
+  gem 'dotenv-rails'
+end
```

Comment:
Remove this and add `gem 'dotenv-rails'` this directly to `:development` group as we are not doing testing.

Reply:
Hi I have added them directly to `:development`. thank you.